src/tooltool/client/tox.ini

```diff
@@ -1,11 +1,15 @@
 # Unified test environment.
 [tox]
-envlist = py27
+envlist = py27, py37
+#envlist = py37
+#envlist = py27
```

Comment:
Nit: Remove these commented lines

Reply:
oh right. that i was using until i figured out `-e` option of tox :)